ts/Data/DataTable.ts

```diff
@@ -1417,6 +1446,67 @@ class DataTable implements DataEvent.Emitter {
         });
     }
 
+    /**
+     * Sets the row key column. This column is invisible and the cells
+     * serve as identifiers to the rows they are contained in. Accessing
+     * rows by keys instead of indexes is necessary in cases where rows
+     * are rearranged by a DataModifier (e.g. SortModifier or RangeModifier).
+     *
+     * @function Highcharts.DataTable#setRowKeysColumn
+     *
+     * @param {number} nRows
+     * Number of rows to add to the column.
+     *
+     */
+    public setRowKeysColumn(nRows: number): void {
+        const id = this.rowKeysId;
+        if (id) {
+            this.columns[id] = [];
+            const keysArray = this.columns[id];
+
+            for (let i = 0; i < nRows; i++) {
+                keysArray.push(id + '_' + String(i));
```

Comment:
When you concat `i` with `'_'` its always a string, so you don't need to use `String(i)`